### DISCUSSION TITLE
Wemos D1 mini and DSC PC1550 Clasic

### DISCUSSION THREAD

**Discussion by pansament** [Dilbert66/esphome-dsckeybus#100] [General]
Is this project working on PC1550 clasic series?

I did wireing and flash D1 mini board. Connectet to alarm board but no comunication between D1 and HomeAssistant.

Can anyone guide me? Or if someone managed to do this project on Clasic Series of DSC alarm ?

**Comment by Dilbert66** [comment]
Sorry no, it does not support the dsc classic. See this repository for dsc classic support: https://github.com/taligentx/dscKeybusInterface

**Comment by pansament** [comment]
Thank you for promt response! 
I`ll try with taligentx Project.
